txtBoiler/Index.txt

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
On my computer I had to change the file name to start with the lowercase, otherwise it was throwing an error